.eslintrc.cjs

```diff
@@ -6,11 +6,25 @@ module.exports = {
     "plugin:@typescript-eslint/recommended",
     "plugin:react-hooks/recommended",
     "plugin:storybook/recommended",
+    "plugin:tailwindcss/recommended",
   ],
   ignorePatterns: ["dist", ".eslintrc.cjs"],
   parser: "@typescript-eslint/parser",
   plugins: ["react-refresh"],
   rules: {
+    "tailwindcss/no-custom-classname": "off",
```

Comment:
noticed this in your other PR as well, whats the reason for keeping this off? it seems to catch a few legitimate issues at this point in time and might be worth having.

Reply:
i arbitrarily added this as to not limit non-tw classNames (if any), i don't think its as important as classname-order or class duplication.

Reply:
I'm not so sure... this means that any refactors (changing config) or typos are not caught, the only guardrail is the intellisense. This essentially allows breaking styles by config changes and using classes that never existed in the first place freely.

for instance, right now, the class `font-mono` is used but actually at some point got [overridden in the config](https://github.com/zmkfirmware/zmk-studio/blob/main/tailwind.config.js#L28-L30)

this could be a followup effort to reduce changes here. The default config for this rule is pretty lax and [by default allows classes that end up in css files](https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/master/docs/rules/no-custom-classname.md#cssfiles-default-css-node_modules--dist-build)